modules/kernel/src/main/scala/org/tessellation/kernel/StateChannelContext.scala

```diff
@@ -11,6 +12,8 @@ trait StateChannelContext[F[_]] {
     hypergraphContext: HypergraphContext[F]
   ): F[Cell[F, StackF, Ω, Either[CellError, Ω], Ω]]
 
+  def interpretOutput(output: Ω): F[StateChannelOutput]
```

Comment:
That's very restricting and I guess only valid in case of L0 Cells. Was that the intention? I mean if we use `interpretOutput` on L1 it needs to have a different definition as not every output will converge into a `Snapshot`.

Reply:
True. I removed it and I just pass output as it is from Cell.